com_redhat_kdump/gui/spokes/kdump.glade

```diff
@@ -197,6 +197,17 @@
                             <property name="top-attach">0</property>
                           </packing>
                         </child>
+                        <child>
+                          <object class="GtkLabel" id="rangeReservedMemMB">
```

Comment:
How about adding a suffix like Label to rangeReservedMemMB to make the code more readable?